kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KafkaSteps.java

```diff
@@ -38,6 +47,35 @@ public static void createTopic(String clusterName, String topicName, String name
                 KafkaTopicTemplates.defaultTopic(namespace, clusterName, topicName, partitions, replicas, minIsr).build());
     }
 
+    /**
+     * Create topic using test clients.
+     *
+     * @param topicName the topic name
+     * @param bootstrap the bootstrap
+     * @param partitions the partitions
+     * @param replicas the replicas
+     */
+    public static void createTopicTestClient(String deployNamespace, String topicName, String bootstrap, int partitions, int replicas) {
+        String podName = KafkaUtils.adminTestClient(deployNamespace, bootstrap);
+        String command = "admin-client topic create --bootstrap-server=" + bootstrap + " --topic=" + topicName + " --topic-partitions=" + partitions +
+                " --topic-rep-factor=" + replicas;
+
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        var exitCode = kubeClient().getClient().pods().inNamespace(deployNamespace).withName(podName)
+                .writingOutput(baos)
+                .writingError(baos)
```

Comment:
Do we really want to have both STD out and STD err in the same stream? feels like that is liable to make things harder to read in the event of errors.

Reply:
done. Thanks!